test/README.md

```diff
@@ -1,93 +1,62 @@
 # Testing Strategy
 
-## Overview
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Unit Testing](#unit-testing)
+- [End to End Testing (System Tests)](#end-to-end-testing-system-tests)
```

Comment:
I think this naming scheme could be clearer, why not just call them what they're called in the code? System Tests and E2E Tests

Reply:
Also quick e2e might be confusing since you can easily run a single heartbeat system test which would be faster than running the e2e tests. I appreciate currently the e2e tests are faster if you run them all but it isn't a fundamental feature of the style of tests.

If we run CCF networks locally in parallel which we might add then system tests would be faster than e2e tests, so the name could also end up being misleading